aws/resource_aws_ses_event_destination_test.go

```diff
@@ -4,24 +4,21 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ses"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
 func TestAccAWSSESEventDestination_basic(t *testing.T) {
```

Comment:
It appears this resource notes import support, however the import testing is not present, which would have caught this issue as well:

```
=== CONT  TestAccAWSSESEventDestination_basic
    TestAccAWSSESEventDestination_basic: testing.go:684: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.

        (map[string]string) {
        }


        (map[string]string) (len=13) {
         (string) (len=24) "cloudwatch_destination.#": (string) (len=1) "2",
         (string) (len=47) "cloudwatch_destination.3035139398.default_value": (string) (len=7) "default",
         (string) (len=48) "cloudwatch_destination.3035139398.dimension_name": (string) (len=13) "ses:source-ip",
         (string) (len=46) "cloudwatch_destination.3035139398.value_source": (string) (len=10) "messageTag",
         (string) (len=46) "cloudwatch_destination.996466118.default_value": (string) (len=7) "default",
         (string) (len=47) "cloudwatch_destination.996466118.dimension_name": (string) (len=9) "dimension",
         (string) (len=45) "cloudwatch_destination.996466118.value_source": (string) (len=11) "emailHeader",
         (string) (len=22) "configuration_set_name": (string) (len=33) "tf_acc_cfg_ses_event_dst_stnw0jtp",
         (string) (len=7) "enabled": (string) (len=4) "true",
         (string) (len=16) "matching_types.#": (string) (len=1) "2",
         (string) (len=25) "matching_types.2185032793": (string) (len=6) "bounce",
         (string) (len=25) "matching_types.2809346765": (string) (len=4) "send",
         (string) (len=4) "name": (string) (len=36) "tf_acc_event_dst_cloudwatch_stnw0jtp"
        }
```

Can you please add the `ImportStateVerify` testing for this?

```go
			{
				ResourceName:      cloudwatchDestinationResourceName,
				ImportState:       true,
				ImportStateVerify: true,
			},
			{
				ResourceName:      kinesisDestinationResourceName,
				ImportState:       true,
				ImportStateVerify: true,
			},
			{
				ResourceName:      snsDestinationResourceName,
				ImportState:       true,
				ImportStateVerify: true,
			},
```

Reply:
Will do.
Something more needs to be fixed:

```console
--- FAIL: TestAccAWSSESEventDestination_basic (123.80s)
    testing.go:684: Step 1 error: error reading SES Configuration Set Event Destination (tf-acc-test-9116975414862209134): InvalidParameterValue: The configuration set name must be specified.
        	status code: 400, request id: 1c004c98-a37f-46b8-b276-b5393289b3fe
```

Reply:
Need to specify the configuration set name and event destination name when importing.

Reply:
Done.